lib/device.js

```diff
@@ -251,6 +256,169 @@ const util = require('util'),
                 }
                 return returnTxt;
             }
+        },
+        /**
+         * get screen capture of the given device
+         * @property options {String} device, display, outputPath
+         */
+        captureScreen: function(options, next) {
+            log.info("device#capture#captureScreen()");
+            if (typeof next !== 'function') {
+                throw new Error('Missing completion callback (next=' + util.inspect(next) + ')');
+            }
+            options = options || {};
+            async.series([
+                _makeSession,
+                _makeCaptureOption,
+                _captureFile,
+                _copyToLocal
+            ],  function(err, results) {
+                log.verbose("device#capture()", "err: ", err, "results:", results);
+                const resultTxt = "Create " + chalk.green(options.captureFileName) + " to " + options.destinationPath +"\nSuccess";
+
+                // clean up /tmp/capture directory in target device
+                if (options.targetCaptureDir) {
+                    _removeCaptureDir(function finish (removeErr){
+                        next(err ? err : removeErr, resultTxt);
+                    });
+                } else {
+                    next(err, resultTxt);
+                }
+            });
+
+            function _makeSession(next) {
+                options.nReplies = 1; // -n 1
+                options.session = new novacom.Session(options.device, next);
+            }
+
+            function _makeCaptureOption(next) {
+                log.info("device#capture#_makeCaptureOption()");
+                let captureFileName = "",
+                    destinationPath = "",
+                    captureFormat = "PNG"; // PNG is default format
+
+                if (options.outputPath === null){
+                    captureFileName = _createDateFileName();
+                    destinationPath = path.resolve('.');
+                } else {
+                    // Get directory path, file path, file Extenstion from given path
+                    const parseDirPath = path.dirname(options.outputPath),
+                        parseBase = path.parse(options.outputPath).base;
+
+                    let parseExt = path.parse(options.outputPath).ext;
+                    parseExt = parseExt.split('.').pop();
```

Comment:
How about use path.extname()?
https://nodejs.org/api/path.html#path_path_extname_path

Reply:
extname returns `.bmp`, I want to get only "bmp" without `.`

Reply:
I'll keep it.